library/Ivoz/Provider/Domain/Service/Extension/ExtensionFactory.php

```diff
@@ -42,6 +49,26 @@ public function fromMassProvisioningCsv(
                 ExtensionInterface::ROUTETYPE_USER
             );
 
+        if ($countryCode) {
+            $country = $this->countryRepository
+                ->findOneByCountryCode($countryCode);
```

Comment:
You'll need to use both, CountryPrefix and CountryCode (This one is optional is the csv) ir order to resolve the country.

CSV example:
```
Extension,CountryPrefix,Number,CountryCode
200,+34,944048184,ES
201,+34,944048185,ES
202,+34,944048186
203,+1,944048187
204,+1,944048188,US
```

Countries in db:
```
mysql> select id, code, name_en, countryCode from Countries where countryCode = '+1';
+-----+------+--------------------------------------+-------------+
| id  | code | name_en                              | countryCode |
+-----+------+--------------------------------------+-------------+
|  38 | CA   | Canada                               | +1          |
| 184 | PR   | Puerto Rico                          | +1          |
| 234 | UM   | United States Minor Outlying Islands | +1          |
| 235 | US   | United States of America             | +1          |
+-----+------+--------------------------------------+-------------+
4 rows in set (0,00 sec)
```